src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/begrunnelser/Standardbegrunnelse.kt

```diff
@@ -497,15 +497,15 @@ enum class Standardbegrunnelse : IVedtakBegrunnelse {
         override val vedtakBegrunnelseType = VedtakBegrunnelseType.INNVILGET
         override val sanityApiNavn = "innvilgetForlengetOppholdEtterEttaarigOppholdstillatelse"
     },
-    INNVILGET_FINNMARKSTILLEGG {
+    INNVILGET_FINNMARKSTILLEGG_MED_DATO {
         override val vedtakBegrunnelseType = VedtakBegrunnelseType.INNVILGET
         override val sanityApiNavn = "innvilgetFinnmarkstillegg"
```

Comment:
Endre gjerne api-navnet også, så kan vi rette opp i sanity når denne er merget
```suggestion
        override val sanityApiNavn = "innvilgetFinnmarkstilleggMedDato"
```